packages/app/src/app/pages/Sandbox/Editor/Workspace/items/Deployment/elements.ts

```diff
@@ -0,0 +1,133 @@
+import styled, { css } from 'styled-components';
+
+const mapColorToState = (state: string, theme: any) => {
+  const STARTING = ['DEPLOYING', 'BUILDING', 'INITIALIZING'];
+  const ERROR = ['DEPLOYMENT_ERROR', 'BUILD_ERROR', 'ERROR'];
+  const STARTED = ['BOOTED', 'READY'];
+
+  if (STARTING.includes(state)) return '#fccb7e';
+  if (ERROR.includes(state)) return theme.red;
+  if (STARTED.includes(state)) return theme.green;
+  if (state === 'FROZEN') return theme.blue;
```

Comment:
@Saeris Add curly braces (`{}`) to make this more clear

```suggestion
  if (state === 'FROZEN') {
    return theme.blue;
  }
```